code/datums/wounds/burns.dm

```diff
@@ -164,18 +164,34 @@
 	. = ..()
 	. += "<div class='ml-3'>"
 
+	. += "Wound Bed Damage: [span_deadsay("[flesh_damage]")]\n"
+	if(flesh_healing > 0)
+		. += "Positive signs of healing in the flesh damage, rate: [span_green("[round(flesh_healing, 0.01)]%")]\n"
+
 	if(infestation <= sanitization && flesh_damage <= flesh_healing)
-		. += "No further treatment required: Burns will heal shortly."
+		. += "[span_green("No further treatment required: Burns will heal shortly.")]"
 	else
-		switch(infestation)
-			if(WOUND_INFECTION_MODERATE to WOUND_INFECTION_SEVERE)
-				. += "Infection Level: Moderate\n"
-			if(WOUND_INFECTION_SEVERE to WOUND_INFECTION_CRITICAL)
-				. += "Infection Level: Severe\n"
-			if(WOUND_INFECTION_CRITICAL to WOUND_INFECTION_SEPTIC)
-				. += "Infection Level: <span class='deadsay'>CRITICAL</span>\n"
-			if(WOUND_INFECTION_SEPTIC to INFINITY)
-				. += "Infection Level: <span class='deadsay'>LOSS IMMINENT</span>\n"
+		if(infestation > 0)
+			. += "Current Sanitization Effect: [span_brass("[round(sanitization/infestation, 0.01)]%")]\n"
```

Comment:
```suggestion
			. += "Current Sanitization Effect: [span_brass("[round(sanitization/infestation*100, 0.01)]%")]\n"
```
So it shows up as a % instead of fraction